src/js/bootstrap-slider.js

```diff
@@ -1314,6 +1314,24 @@ const windowIsDefined = (typeof window === "object");
 								}
 								this.tickLabelContainer.style[styleMargin] = this.sliderElem.offsetWidth / 2 * -1 + 'px';
 							}
+
+							/* Set class labels to indicate tick labels are in the selection or selected */
+							this._removeClass(this.tickLabels[i], 'label-in-selection label-is-selection');
+							if (!this.options.range) {
+								if (this.options.selection === 'after' && percentage >= positionPercentages[0]) {
+									this._addClass(this.tickLabels[i], 'label-in-selection');
+								} else if (this.options.selection === 'before' && percentage <= positionPercentages[0]) {
```

Comment:
anyway we can add some unit tests for this? something as simple as ensuring the tick label elements that get rendered have the expected class tag?

Reply:
thanks for the detailed example, i'll give it a shot soon. been busy as we're launching a new product line, featuring bootstrap-slider!

this is a priority, since we don't want our app dependent on my fork; just gotta clear out some time to put in some work

Reply:
@duggi thats awesome! if its possible, we would love to see a screenshot or GIF of the slider in your new product! Its always exciting to see how the project is being used :)

Reply:
screenshot in my initial comment ... repasted at bottom.

you can also see it live [here](https://www.guidevine.com/financial-advisors/retirement-planning), click on "Free Consultation" for any advisor. Note this impl is single value only, not range. also uses different styles.

(please don't actually submit a request)

--

![image](https://user-images.githubusercontent.com/370924/30177835-daa03198-93bb-11e7-824f-80eab709852f.png)